src/Serializer/SubjectNormalizer.php

```diff
@@ -95,6 +108,13 @@ public function normalize($object, $format = null, array $context = []) : array
             if ($object->getImpactStatement()) {
                 $data['impactStatement'] = $object->getImpactStatement();
             }
+
+            if ($object->getAimsAndScope()->notEmpty()) {
+                $data['aimsAndScope'] = $object->getAimsAndScope()
+                    ->map(function (Block $block) use ($format, $context) {
```

Comment:
I saw in https://github.com/elifesciences/api-raml/pull/214 that this should be a list of paragraphs, so you may be able to use `Paragraph` in place of `Block` across this class. Not sure if `Block` was put there to make the code more general and consistent with the other serializers.

Reply:
I have adjusted this to expect only `Block\Paragraph`.